themes/apachecon/layouts/search/section.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE HTML>
+
+<html>
+{{ partial "header.html" . }}
+
+<body>
+
+  <!-- Header Wrapper -->
+  <div class="wrapper style1">
+  
+      {{ partial "smallpageheader.html" . }}
+  
+      <!-- Main -->
+      <div id="main" class="wrapper style4">
+  
+          <!-- Content -->
+          <div id="content" class="container">
+            <header class="major">
+              <h2>{{i18n "search"}}</h2>
+          </header>
+            <section class="track">
+            <div class="search" οnlοad="getTestXmlValue();">
+              <input type="text" placeholder="{{i18n "searchContent"}}" id="search-key" />
+              <input type="hidden" name="search_hidden" value="{{i18n "search"}}" id="search_hidden" />
+              <button onclick="search()">
+                <svg t="1583982313567" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="1271"
```

Comment:
Can you check if this image has the copyright?
Can we use it directly here?

Reply:
I will upload a picture of the search icon again.